core/src/main/java/eu/fasten/core/plugins/GraphDBConnector.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.fasten.core.plugins;
+
+import org.rocksdb.RocksDBException;
+
+/**
+ * A plug-in that needs to access graph database (RocksDB)
+ */
+public interface GraphDBConnector extends FastenPlugin {
+
+    /**
+     * Sets location where RocksDB database is stored
+     *
+     * @param dir Path to database directory
+     */
+    void setRocksDbDir(String dir) throws RocksDBException;
```

Comment:
Moreover, I remember we discussed that we don't do checked exceptions.

Reply:
I was a bit confused about error handling. When initializing RocksDao it can throw `RocksDBException`. Should the try/catch block be in the Graph Plugin class or in the server when initializing the plugin?

Reply:
You should wrap RocksDBException into a Runtime exception and let if flow to the server.

Reply:
Done